doc/rtd/development/datasource_creation.rst

```diff
@@ -158,10 +158,11 @@ packaging configuration.
 Add documentation for your datasource
 -------------------------------------
 
-You should add a new file in
-:file:`doc/rtd/reference/datasources/<cloudplatform>.rst`
-and reference it in
-:file:`doc/rtd/reference/datasources.rst`
+You should update the following docs:
+1. Add a new file in :file:`doc/rtd/reference/datasources/<cloudplatform>.rst`
+2. Reference `<cloudplatform>.rst` in :file:`doc/rtd/reference/datasources.rst`
+3. Add an alphebetized dsname entry in representing your datasource
+:file:`doc/rtd/reference/datasource_dsname_map.rst`
```

Comment:
should there be a more direct call to action here to use the command found in `doc/rtd/reference/datasource_dsname_map.rst` to generate the datasource list? Or is it just easier to tell the user to simply insert their new DS name in the list alphabetically (like you currently have)?

Reply:
It shouldn't need to be regenerated each time, so I think the instructions here make the most sense.